### DISCUSSION TITLE
How does world->raycast() work?

### DISCUSSION THREAD

**Discussion by anugentc** [DanielChappuis/reactphysics3d#322] [Q&A] [answered]
So what I want to do is to have a raycast into the entire world that returns a point of contact. I got world->raycast() to work and print out the contact point, but I am unsure of what to do from there. 
The method is void, so nothing is returned, and I can't return a vector or anything. 

Am I on the right track, or is there something else I need to do for it?

**Comment by DanielChappuis** [comment, marked as answer]
Hello,

You should take a look at the documentation about raycasting [here](https://reactphysics3d.com/usermanual.html#x1-6900013).

I hope this helps.

**Reply by anugentc** [reply]
oh, thank you!
